corehq/apps/app_manager/static/app_manager/json/commcare-profile-settings.yml

```diff
@@ -328,18 +220,6 @@
   since: "2.4"
   force: true
 
-- name: "Login Buttons"
-  description: "Choose to label the login buttons with Icons or Text"
-  id: "cc-login-images"
-  values:
-    - ['Yes', 'Icons']
-    - ['No', 'Text']
-  default: "No"
-  contingent_default: [{"condition": "{features.sense}='true'", "value": "Yes"}]
-  values_txt: "This value will set whether the login screen uses customizable icons for login and demo mode options or uses the standard buttons with labels."
-  since: "2.8"
-  force: true
```

Comment:
how do you know which of these settings only apply to J2ME?

Reply:
That was based on corehq/apps/app_manager/static/app_manager/json/commcare-settings-layout.yml and I checked in with Shubham. There was one of those that I moved to the android section instead of removing it.

Reply:
I've also verified these by looking at the commcare-core and commcare-android code